crates/aptos-protos/proto/aptos/tokens/v1/tokens.proto

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package aptos.tokens.v1;
+
+
+message Tokens {
+  uint64 block_height = 1;
+  uint32 chain_id = 2;
+  repeated Token tokens = 3;
+  repeated TokenData token_datas = 4;
+  repeated CollectionData collection_datas = 5;
+}
+
+message Token {
+  TokenId token_id = 1;
+  uint64 transaction_version = 2;
```

Comment:
just curious if we modify the token's amount what would be the transaction_version?

Reply:
Each transaction version will be recorded as a separate row. So we'd have 2 rows w/ different token amounts.